src/components/character-avatar/character-avatar.tsx

```diff
@@ -20,12 +28,38 @@ export const CharacterAvatar: FC<Props> = ({
   isBot,
   onSelectPlayer,
 }) => {
+  const animation = getCharacterAnimation(character.name, isBot, isSelected);
+  const [segments, setSegments] = useState<number[][]>([[]]);
+
+  useEffect(() => {
+    if (isBot) {
+      const animationSegments = isSelected
+        ? BOT_BUSTED_ANIMATION_SEGMENT
+        : BOT_WIN_ANIMATION_SEGMENT;
+      setSegments(animationSegments);
+    }
+  }, [isBot, isSelected]);
+
   return (
-    <Stack
-      sx={styles.avatar(stage, isSelected, isBot)}
-      onClick={onSelectPlayer}
-    >
-      {getCharacterAvatar(character.name)}
-    </Stack>
+    <>
+      {stage === "voting" && (
+        <Stack
+          sx={styles.avatar(stage, isSelected, isBot)}
+          onClick={onSelectPlayer}
+        >
+          {getCharacterAvatar(character.name)}
+        </Stack>
+      )}
+      {stage !== "voting" && (
+        <Stack sx={styles.animation(stage, isSelected, isBot)}>
+          <AnimationPlayer
+            animationData={animation}
+            segments={segments}
+            play
+            loop
+          />
+        </Stack>
+      )}
+    </>
```

Comment:
Use a ternary instead of `&&` and perform the check on the stack children and the onClick function instead of doing it for everything since Stack is in common for both cases